hugr-py/src/hugr/serialization/__init__.py

```diff
@@ -1,3 +0,0 @@
-from .serial_hugr import SerialHugr
-
-__all__ = ["SerialHugr"]
```

Comment:
I'm not quite sure what's going on here, I'm surprised that a lint would tell us to change this interface?

Reply:
ah yeah sorry, this is because some of the import changes caused a circular import problem. This was my chosen solution - we don't rexport through init anywhere else and the serialised hugr should be an internal thing now anyway